CMakeLists.txt

```diff
@@ -176,14 +176,14 @@ install(
 set(API_JSON_FILE ${PROJECT_BINARY_DIR}/unified_runtime.json)
 
 # Generate source from the specification
-add_custom_target(generate-code
+add_custom_target(generate-code USES_TERMINAL
```

Comment:
why do we need USES_TERMINAL?

Reply:
This is not essential but I find it makes iterating on spec changes faster.